wire-ios-sync-engine/Source/Use cases/ToggleMessageReactionUseCase.swift

```diff
@@ -31,9 +32,14 @@ public protocol ToggleMessageReactionUseCaseProtocol {
 public struct ToggleMessageReactionUseCase: ToggleMessageReactionUseCaseProtocol {
 
     weak var analyticsEventTracker: (any AnalyticsEventTracker)?
+    var analyticsLogger: WireLogger
```

Comment:
Here, and in the other message use cases, I see this logger that isn't being used. I think we can also clean them up if you don't intend to use them.